ci-operator/step-registry/openshift-logging/enable-cluster-logging/openshift-logging-enable-cluster-logging-commands.sh

```diff
@@ -0,0 +1,582 @@
+#!/bin/bash
+set -o nounset
+
+storage_class=""
+minio_secret=""
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+# setup proxy
+if test -f "${SHARED_DIR}/proxy-conf.sh"
+then
+    source "${SHARED_DIR}/proxy-conf.sh"
+fi
+
+
+### common function ###
+function prepare_namespace(){
+    echo "Prepare namespace $1"
+    if oc get project $1 -o name; then
+        echo "use existing namespace $1"
+    else
+        echo "create namespace $1"
+        oc adm new-project $1  || exit 1
+    fi
+    oc label ns $1 openshift.io/cluster-monitoring="true"
+    oc label ns $1 openshift.io/log-alerting="true"
+}
+
+function are_operators_ready(){
+    echo " Wait the operators ready up to 1 minutes"
```

Comment:
Is 1 minute enough?

Reply:
BTW, do we really need this check?
I remembered the step `install-operators` already have some checks after the installation.